.gitignore

```diff
@@ -1,5 +1,6 @@
 **/build
+**/coverage
```

Comment:
Maybe selecting coverage folders directly would be more suitable, since there are only two of them and if we want to use that folder name for smth else

Reply:
Good call, I will adjust this in when I resolve the conflicts